figure_notebooks/supp_analyze_journalist_data/journalist_analysis.md

```diff
@@ -0,0 +1,389 @@
+    ggtitle("Est. Proportion Written Articles by Gender in Nature") + 
+    ylim(c(0, 1)) +
+    geom_hline(yintercept=0.5, color="red") +
+    scale_fill_manual(values=c("#E69F00", "#56B4E9")) +
+    theme(legend.position="bottom")
+
+ggsave(file.path(proj_dir,  "/figure_notebooks/supp_analyze_journalist_data/tmp_files/journo_gender_gg.pdf"),
+       journo_gender_gg, width = 6, height = 5, units = "in", device = "pdf")
+
+journo_gender_type_m_gg = 
+    ggplot(type_df_m, 
+      aes(x=as.numeric(year), y=mean,
+                          ymin=bottom_CI, ymax=top_CI,
+                          fill=corpus)) +
+    geom_point() + geom_ribbon(alpha=0.5) + geom_line(alpha=0.5) + theme_bw() + 
+    xlab("Year of Article") + ylab("Male Percentage") +
+    ggtitle("Est. Proportion Male Written Articles in Nature") + 
+    ylim(c(0, 1)) +
+    geom_hline(yintercept=0.5, color="red") +
+    theme(legend.position="bottom")
+
+ggsave(file.path(proj_dir,  "/figure_notebooks/supp_analyze_journalist_data/tmp_files/journo_gender_type_m_gg.pdf"),
+       journo_gender_type_m_gg, width = 6, height = 5, units = "in", device = "pdf")
+
+
+journo_gender_type_f_gg = 
+    ggplot(type_df_f, 
+      aes(x=as.numeric(year), y=mean,
+                          ymin=bottom_CI, ymax=top_CI,
+                          fill=corpus)) +
+    geom_point() + geom_ribbon(alpha=0.5) + geom_line(alpha=0.5) + theme_bw() + 
+    xlab("Year of Article") + ylab("Male Percentage") +
+    ggtitle("Est. Proportion Female Written Articles in Nature") + 
+    ylim(c(0, 1)) +
+    geom_hline(yintercept=0.5, color="red") +
+    theme(legend.position="bottom")
+
+ggsave(file.path(proj_dir,  "/figure_notebooks/supp_analyze_journalist_data/tmp_files/journo_gender_type_f_gg.pdf"),
+       journo_gender_type_f_gg, width = 6, height = 5, units = "in", device = "pdf")
+
+
+
+#### plot the overview of name origin
+journo_name_origin_gg = ggplot(journo_origin_df, aes(x=as.numeric(year), y=mean,
+                          ymin=bottom_CI, ymax=top_CI,
+                          fill=name_origin, label=label)) +
+    geom_point() + geom_ribbon(alpha=0.5) + geom_line(alpha=0.5) +
+    theme_bw() + geom_text_repel() + xlim(c(2005,2021))  +
+    xlab("Year of Article") + ylab("Proportion of Journalist Citations") +
+    ggtitle("Est. proportion of articles written by specific name origin ") + 
+    scale_fill_brewer(palette="Set2") +
+    theme(legend.position = "none")
+
+ggsave(file.path(proj_dir,  "/figure_notebooks/supp_analyze_journalist_data/tmp_files/journo_name_origin_gg.pdf"),
+       journo_name_origin_gg, width = 7, height = 5, units = "in", device = "pdf")
+
+journo_gender_gg
+```
+
+![](journalist_analysis_files/figure-markdown_github/make_citation_gg-1.png)
+
+``` r
+journo_gender_type_m_gg
+```
+
+![](journalist_analysis_files/figure-markdown_github/make_citation_gg-2.png)
+
+``` r
+journo_gender_type_f_gg
+```
+
+![](journalist_analysis_files/figure-markdown_github/make_citation_gg-3.png)
```

Comment:
Are the areas showing confidence intervals for each proportion estimate? Also why does the technology-feature category have such a wide area?

Reply:
yeah, it's because there are so few articles in the first place, dividing it by author gender makes even fewer articles. Then in the 4 years where there is no confidence interval, it is because all articles (which are < 10) were written by women.

Reply:
I added the raw count tables to give more context: https://github.com/greenelab/nature_news_disparities/pull/33/commits/8804f50e9b1b8dab9a83e744f36add1a09eb6e30